pkg/helm/helm.go

```diff
@@ -145,6 +149,7 @@ func (r *Impl) Install(chart *chart.Chart, kubeNamespace string, releaseName str
 	install := action.NewInstall(cfg)
 	install.ReleaseName = releaseName
 	install.Namespace = kubeNamespace
+	install.Wait = true
```

Comment:
How does this work when used from a reconcile loop? Since it's a long running process is it more likely to have overlapping installs now? Are we dealing with it in the blueprint reconciler?

Reply:
Yes, but not in this PR, thus I'm removing the wait argument for now.